app/controllers/application_controller.rb

```diff
@@ -61,6 +61,57 @@ def set_app_info_headers
     headers['X-GitHub-Repository'] = AppInfo::GITHUB_REPO
   end
 
+  def handle_error(error)
+    Common::Client::ErrorHandler.handle(error)
+  end
+
+  def render_error(exc)
+    render json: { errors: exc.errors }, status: exc.status
+  end
+
+  # @note Later rescue_from statements take precedence and so we must take care to respect order
+  rescue_from StandardError do |error|
+    exc = Common::Exceptions::InternalServerError.new(error)
+    handle_error(exc)
+    render json: { errors: exc.errors }, status: exc.status_code
```

Comment:
Could we extend the render_error method to include json rendering for the exc.status_code case (lines 76 and 82 of this PR)?